### DISCUSSION TITLE
Masking Buyer Contact details from logistics provider

### DISCUSSION THREAD

**Discussion by BLR-0118** [beckn/protocol-specifications#220] [Core Specification]
Some of our participants have a policy where they aren't allowed to share their customer phone details with the logistics provider. Everything is masked and the logistics provider can reach out to customers only through their IVR. How is this typically handled?

**Comment by BLR-0118** [comment]
One of the ways to do this may be through Cloud Telephony where logistics provider can send a request for Cloud Telephony number in /support, in response to which the counterparty provides this number in /on-support and the logistics provider can initiate Cloud Calling.
Are there other ways of doing this?

**Comment by BLR-0118** [comment]
@mherle @pkapustin

**Reply by mherle** [reply]
1 - A Delivery Executive calls a preconfigured IVR number and then dials a PIN/ORDERID to connect
2 - A Conf call is done between the customer and Delivery Executive is done (could be initiated via seller or via logistics: but in any case, one party will share a number with the other party - either the customer's or the delivery executive's)
3 - Share the actual numbers between the systems

Both 1 and 2 will cost money and may not be acceptable to all.
